### PR TITLE
Individual ben is address uk

### PR DESCRIPTION
**Pull request by jassalrichy**
Individual ben is address uk

### CODE REVIEW

File: test/navigation/navigators/BeneficiaryRoutes.scala [hunk 1]
```diff
@@ -193,6 +193,24 @@ trait BeneficiaryRoutes {
...
+    "go to IndividualBeneficiaryAddressUKPage from IndividualBeneficiaryAddressUKYesNoPage when user answers yes" in {
+      forAll(arbitrary[UserAnswers]) {
+        userAnswers =>
+          val answers = userAnswers.set(IndividualBeneficiaryAddressUKYesNoPage(indexForBeneficiary), value = true).success.value
+          navigator.nextPage(IndividualBeneficiaryAddressUKYesNoPage(indexForBeneficiary), NormalMode)(answers)
+            .mustBe(routes.IndividualBeneficiaryAddressUKController.onPageLoad(NormalMode, indexForBeneficiary))
+      }
+    }
+
+    "go to IndividualBeneficiaryAddressUKPage from IndividualBeneficiaryAddressUKPage when user answers no" in {
```
Comment (adamconder) [inline test/navigation/navigators/BeneficiaryRoutes.scala]: It should be `go to IndividualBeneficiaryAddressUKYesNoPage`?

File: test/controllers/IndividualBeneficiaryAddressUKYesNoControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+  val formProvider = new IndividualBeneficiaryAddressUKYesNoFormProvider()
+  val form = formProvider()
+  val index: Int = 0
+  val name = FullName("first name", None, "Last name")
+
+  lazy val individualBeneficiaryAddressUKYesNoRoute = routes.IndividualBeneficiaryAddressUKYesNoController.onPageLoad(NormalMode, index).url
+
+  "IndividualBeneficiaryAddressUKYesNo Controller" must {
+
```
Comment (adamconder) [inline test/controllers/IndividualBeneficiaryAddressUKYesNoControllerSpec.scala]: Do you require a unit test for when the required answer is not present in mongo?

File: test/navigation/navigators/BeneficiaryRoutes.scala (hunk 1 above)
Comment (jassalrichy) [reply test/navigation/navigators/BeneficiaryRoutes.scala]: Amended

File: test/controllers/IndividualBeneficiaryAddressUKYesNoControllerSpec.scala (hunk 2 above)
Comment (jassalrichy) [reply test/controllers/IndividualBeneficiaryAddressUKYesNoControllerSpec.scala]: Yes, added.

**Pull request merged**